### PR TITLE
fix ssr url search params bug

### PR DESCRIPTION
**Pull request by FredKSchott**
## Changes

- Fixes the issue identified in #3057

## Testing

- Added.

## Docs

- N/A

### CODE REVIEW

File: packages/astro/src/cli/index.ts [hunk 1]
```diff
@@ -49,7 +49,6 @@ function printAstroHelp() {
 			['--config <path>', 'Specify the path to the Astro config file.'],
 			['--root <path>', 'Specify the path to the project root folder.'],
 			['--legacy-build', 'Use the build strategy prior to 0.24.0'],
-			['--experimental-ssr', 'Enable SSR compilation fot 3rd-party adapters.'],
```
Comment (JuanM04) [inline packages/astro/src/cli/index.ts:52]: Oops, it seems that some unwanted changes got leaked

Comment (JuanM04) [reply packages/astro/src/cli/index.ts:52]: Oh, sorry, I didn't know about it

Comment (FredKSchott) [reply packages/astro/src/cli/index.ts:52]: No problem! I should have added a comment here for reviewers

**Comment by FredKSchott** [comment]
Self-merging as steward since core team is out today, and this is small.

**Pull request merged**